README.md

```diff
@@ -49,7 +54,7 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+guests.push residents.delete_at(1)
 ```
```

Comment:
The computer doesn't know what to do with `guests` and `residents` on their own. They only exists as keys within the hash `town`, so need to be referenced as such.

One possible solution would be...
```
belle = town[:residents][1]
town[:residents].delete(belle)
town[:castle][:guests]<< belle
```